locale/sv_SE/editor.xml

```diff
@@ -147,4 +147,5 @@
 	<message key="editor.submission.editorial.finalDraftDescription"><![CDATA[Slutgiltiga utkastfiler i den här listan görs om till manusredigerade versioner av en manusredaktör och laddas upp nedan under <span class="pkp_help_title">Manusredigering</span>.]]></message>
 	<message key="editor.submission.proof.manageProofFilesDescription">Alla filer som redan laddats upp i något av processens steg kan läggas till i listan för Korrekturläsning genom att kryssa i rutan "Inkludera" nedan och klicka på Sök: alla tillgängliga filer listas och kan väljas för inkludering.</message>
 	<message key="editor.publicIdentificationExistsForTheSameType">Den publika identifikatorn '{$publicIdentifier}' existerar redan för ett annat objekt av samma typ. already exists for another object of the same type. Vänligen välj unika identifikatorer inom tidskriften.</message>
+  <message key="editor.article.cover">Omslag</message>
```

Comment:
hmmm... this seems that it should be "Omslagsbild" ?
https://github.com/pkp/ojs/commit/e0d97bc71491a5a9591eab6815de3572577c3ff8#diff-bcc04c6fa95f52b0ccad9502f1cd1c11L211